pkgs/development/python-modules/korean-lunar-calendar/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv
```

Comment:
```suggestion
{ lib
```
Maybe just use lib since ``stdenv`` isn't actually used?

Reply:
I heavily prefer keeping `stdenv` around (and wish everyone did) because I'm forever having to dig into `if stdenv.isDarwin...`, `if stdenv.cc.isClang...` and the like.